### PR TITLE
Renamed column.

### PR DESCRIPTION
**Pull request by ruschein**

### CODE REVIEW

File: cpp/data/installer/ub_tools.sql [hunk 1]
```diff
@@ -40,10 +40,10 @@ CREATE TABLE zeder_journals (
 -- Table to be used w/ our validate_harvested_records tool:
 CREATE TABLE metadata_presence_tracer (
     zeder_journal_id INT(11) UNSIGNED,
-    metadata_field_name CHAR(4) NOT NULL,
+    marc_field_tag CHAR(4) NOT NULL,
```
Comment (mtrojan-ub) [inline cpp/data/installer/ub_tools.sql:43]: 3

Comment (ruschein) [reply cpp/data/installer/ub_tools.sql]: Fixed

File: cpp/data/sql_updates/ub_tools.22 [hunk 2]
```diff
@@ -0,0 +1 @@
+ALTER TABLE metadata_presence_tracer RENAME COLUMN metadata_field_name TO marc_field_tag
```
Comment (mtrojan-ub) [inline cpp/data/sql_updates/ub_tools.22:1]: length should be changed to 3 here as well

Comment (ruschein) [reply cpp/data/sql_updates/ub_tools.22:1]: Done!

Comment (mtrojan-ub) [reply cpp/data/sql_updates/ub_tools.22:1]: Multiple Problems:
- Please use MODIFY COLUMN syntax so it works on Ubuntu + CentOS
- Don't forget NOT NULL in ub_tools.22
- Why did you add record_type in this pull request? (it is used nowhere)
- If you add a new column, please also use the AFTER keyword to add it at the right place (else diffs will get more and more confusing)

Comment (mtrojan-ub) [reply cpp/data/sql_updates/ub_tools.22:1]: And please use semicolon after each statement!

Comment (ruschein) [reply cpp/data/sql_updates/ub_tools.22]: Done.

**Pull request merged**